### DISCUSSION TITLE
Looking for assistance - V4 maps

### DISCUSSION THREAD

**Discussion by youca04** [mapsforge/mapsforge#1387] [Q&A] [answered]
Hi all. These are a couple of questions related to my previous question about v4 maps downloads.

Basically, we are not in a position to move to the v5 versions in the short term and we have a need for a v4 map of Pakistan, so:

1. Is there anyone who could assist in producing a v4 map of Pakistan? I am willing to pay for this effort of course.
2. Is there anyone who could assist in setting up an Osmosis build environment to produce v4 maps until we can get the software we use upgraded to suport v5 natively.

Many thanks for any responses.

Regards,
Carl

**Comment by devemux86** [comment, marked as answer]
There is a [detailed step-by-step guide](https://github.com/mapsforge/mapsforge/blob/master/docs/MapCreation.md) on how to create Mapsforge maps.
Or the automatic generation with [mapsforge-creator](https://github.com/mapsforge/mapsforge-creator).

**Comment by tqu-dev** [comment]
>     1. Is there anyone who could assist in producing a v4 map of Pakistan?
I have maps of Pakistan which I can offer you for testing. You can contact me here: https://www.cartograph.eu/v3/contact/

**Comment by PrathameshGujar08** [comment]
> Hi all. These are a couple of questions related to my previous question about v4 maps downloads.
> 
> Basically, we are not in a position to move to the v5 versions in the short term and we have a need for a v4 map of Pakistan, so:
> 
> 1. Is there anyone who could assist in producing a v4 map of Pakistan? I am willing to pay for this effort of course.
> 2. Is there anyone who could assist in setting up an Osmosis build environment to produce v4 maps until we can get the software we use upgraded to suport v5 natively.
> 
> Many thanks for any responses.
> 
> Regards, Carl

Did you find a way to get v4 maps? I am looking for the same for India. If you found someway it would be helpful for me too. If you could share it to me

**Reply by youca04** [reply]
I set up a build server with 128GB of RAM so I can generate this for you. If you can find a way to share your email address with me, I’ll send you a download link. 

My address (remove all underscores)
c_a_r_l_y_o_u_n_g@k_eyco_mm.c_o.u_k

**Reply by PrathameshGujar08** [reply]
Hey There! Thanks for replying. Would you charge for providing the map or something? I have mailed you.

**Reply by youca04** [reply]
I will send you the map free of charge, don't worry. I've started the build of the map - takes many hours to build.
I haven't seen an email from you with an address I can use (only messages via github). One of my email addresses was in the previous reply. You need to mail me at that address after removing all underscores from it.

**Reply by PrathameshGujar08** [reply]
Thank you very much for helping me out. I have mailed you again if you can check
.

**Reply by youca04** [reply]
Got the mail. Will send as soon as build complete.

**Reply by PrathameshGujar08** [reply]
Thanks that will help me alot!

**Reply by PrathameshGujar08** [reply]
Got it in the mail today! Thanks. Would you like to connect on LinkedIn?

**Comment by youca04** [comment]


**Reply by PrathameshGujar08** [reply]
Carl Young on LinkedIN? How can I find you I tried but there are many
